kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/QueryDataServiceBase.java

```diff
@@ -109,30 +117,43 @@ public Object query(String queryName, String mapper, String orderBy, Integer pag
 
         QueryResultMapper<?> resultMapper = QueryMapperRegistry.get().mapperFor(mapper, null);
         QueryContext queryContext = buildQueryContext(page, pageSize);
+
         if (orderBy != null && !orderBy.isEmpty()) {
-            queryContext.setOrderBy(orderBy);
-            queryContext.setAscending(true);
-        }
+            String[] orderBySortOrderItems = orderBy.split(",");
+
+            if (orderBySortOrderItems.length > 1) {
+                if (logger.isDebugEnabled())
+                    logger.debug("-- query() > orderBy clause = " + orderBy);
```

Comment:
the only minor comment is that we don't need to use isDebugEnabled any more using logger.debug("{}") should be good enough